web/styles/compose.css

```diff
@@ -263,14 +263,51 @@
     }
 }
 
-#compose-textarea,
-#preview-message-area-container {
-    grid-area: compose-textarea;
-    /* When the compose box is full screen, this
-       auto height will kick in and make sure that
-       the textarea opens to fill its whole grid
-       area. */
-    height: auto;
+#message-content-container {
+    grid-area: message-content-container;
+    display: grid;
+    grid-template: minmax(0, 1fr) / minmax(0, 1fr) 25px;
```

Comment:
I'd really like to see named grid areas here and the textarea and buttons div placed on them explicitly. I think that will end up being important for when you go to get the formatting buttons in place, as those will likely need to take up the space in the column you've set up for the expand/collapse buttons. Plus it's just good practice in an area like this to use named areas for the sake of contributors who will have no idea how the expand/collapse buttons end up where they do.